packages/rum/test/e2e/index.js

```diff
@@ -24,7 +24,7 @@
  */
 
 import { polyfill } from 'es6-promise'
-import { apmBase } from '../../src'
+import { apmBase } from '@elastic/apm-rum'
```

Comment:
The reason why this has to be changed is the same file has been used to create apm instance inside `rum-react` package https://github.com/elastic/apm-agent-rum-js/blob/28df070ed4001d1e3ee3fcb929364e5960339793/packages/rum-react/test/e2e/index.js#L26

Due to our new bundling one reference would go to `packages/rum/src/index.js` and other would go to `packages/rum/dist/es/index.js` and both would result in two apm instances which would make the test fail. By referencing it with package name we solve this issue since webpack always resolves to `module` field in package.json.